av/container/output.pyi

```diff
@@ -1,5 +1,21 @@
+from fractions import Fraction
+from typing import Sequence
+
+from av.packet import Packet
+from av.stream import Stream
+
 from .core import Container
 
 class OutputContainer(Container):
     def __enter__(self) -> OutputContainer: ...
+    def add_stream(
+        self,
+        codec_name: str,
+        rate: Fraction | int | float,
+        template: Stream,
```

Comment:
Should be `template: Stream | None = None`

Reply:
I see that either codec_name or template is needed. Should I type it as an overload function instead?

Reply:
Yes, overload is appropriate here

Reply:
Sorry I edited the previous comment, I found out that typing as overloading might be impossible